vars/pmebuild.groovy

```diff
@@ -45,14 +51,15 @@ def buildProject(String project, String settingsXmlId, Map<String, Object> build
         executePME(finalProjectName, projectConfig, pmeCliPath, settingsXmlId, variableVersionsMap)
         executeBuildScript(finalProjectName, buildConfig, settingsXmlId, "-DaltDeploymentRepository=local::default::file://${env.WORKSPACE}/deployDirectory")
 
+        def key = projectVariableMap[group + '_' + name]
```

Comment:
what happens if projectVariableMap does not contain the `group + '_' + name` ?

Reply:
```suggestion
```